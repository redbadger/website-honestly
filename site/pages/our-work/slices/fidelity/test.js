import React from 'react';
import { shallow } from 'enzyme';

import FidelitySlice from './index';

describe('pages/our-work/case-study/slices/fidelity', () => {
  it('renders correctly', () => {
    const rendered = shallow(<FidelitySlice />);
    expect(rendered).toMatchSnapshot();
  });
});
