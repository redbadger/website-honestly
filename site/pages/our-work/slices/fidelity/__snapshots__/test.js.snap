// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`pages/our-work/case-study/slices/fidelity renders correctly 1`] = `
<Container
  to="fidelityCaseStudy"
>
  <Text
    layout="leftCol"
  >
    <Header>
      Global design and build project for Fidelity International, delivered in four months
    </Header>
    <Description>
      A website which creates a new industry benchmark for design & usability, and empowers the client to grow the site and their audience, with tools and ways of working.
    </Description>
    <ReadMore />
  </Text>
  <Image
    alt="Illustration of a brief-case carrying man in a suit walking along a path through stylised mountains."
    src="listing.jpg"
  />
</Container>
`;
